src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -124,6 +124,11 @@ public String getCustomMessage() {
         return customMessage;
     }
 
+	public SlackNotifier()
+	{
+		
+	}
+	
     @DataBoundConstructor
     public SlackNotifier(final String teamDomain, final String authToken, final String room, final String buildServerUrl,
```

Comment:
Its a requirement of Groovy, you can go ahead and try it yourself but you will run into an error.